Added drop down for selecting Subject handled

hariharan-mahadevan-10884 opened the pull request.

Commit:
Added dropdown for selecting subject handled

sivaraam committed:
Merge branch 'master' into Select

sivaraam commented on css/light-styles.css:
```diff
@@ -20,7 +20,7 @@ html, body {
 .item-description {
-  font-weight: bold;
+  //font-weight: bold;
   font-size: 125%;
```
Uncomment this

sivaraam commented on css/main.css:
```diff
@@ -183,13 +183,12 @@ html, body {
 table {
-  border:1px solid;
+  border:none;
   border-collapse: collapse;
```
Don't remove the border

sivaraam commented on css/radio-button.css:
```diff
@@ -22,6 +22,7 @@
 .css-radio .css-button .css-button-label-container {
   border: 0.1em solid;
+  border-radius:0.25em;
   margin: 0 1.5em;
```
Remove this line

sivaraam reviewed (changes requested):
Leave the labels as they were before

Commit:
Made changes
1.Brought label style as before
2.Brought table borders back
3.removed border radius for radio buttons

Commit:
Made changes
1.Brought table borders back
2.Removed border radius of radio buttons
3.Changed label style

sivaraam committed:
Merge branch 'master' into Select

sivaraam commented on css/light-styles.css:
```diff
@@ -45,6 +45,21 @@ table th, table td {
+   background-position: 87% center;
+   background-repeat: no-repeat;
+   -webkit-appearance: button;  
+
```
-webkit-appearance works only for a few browsers. See https://developer.mozilla.org/en/docs/Web/CSS/-moz-appearance

sivaraam commented on index.html:
```diff
@@ -206,6 +206,25 @@ <h1 class="title">Seminar Hall Booking</h1>
+					</div>
+				  </div>
+				</div>
 
```
Indentation to be fixed

Commit:
Added -moz-appearance to the drop down style

Commit:
Merge branch 'Select' of https://github.com/harideepan/Seminar into Select

Commit:
added appearance property to drop down

sivaraam commented on css/main.css:
```diff
@@ -471,7 +470,25 @@ table th, table tr , table td{
+     cursor: pointer;
+     text-overflow: ellipsis;
+     white-space: nowrap;
+     width: 18.75em;
```
The options look a little odd. Better to avoid fixing width property. 

Do you wish to take a look at how [Bootstrap](http://getbootstrap.com/) implement select options ? Else, i'll take care of this.


![crop](https://cloud.githubusercontent.com/assets/12448084/25385199/dee1c4a0-29e0-11e7-8c81-b8487cad3f90.png)
